bin/hook-docs/data/actions.json

```diff
@@ -1,6 +1,18 @@
 {
     "$schema": "https://raw.githubusercontent.com/wp-hooks/generator/0.9.0/schema.json",
     "hooks": [
+        {
+            "name": "deprecated_function_run",
+            "file": "Domain/Bootstrap.php",
+            "type": "action",
+            "doc": {
+                "description": "",
+                "long_description": "",
+                "tags": [],
+                "long_description_html": ""
+            },
+            "args": 3
+        },
```

Comment:
Bootstrap file had a syntax error and never actually run, now it's running and it picked up this one.